<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="Born_Day">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Born Day</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Born_Month">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Born Month</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Born_Year">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Born Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Today_Year">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Today Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Today_Month">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Today Month</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Today_Day">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Today Day</string>
    </property>
   </widget>
   <widget class="QLabel" name="Day_info">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Day</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Month_info">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Year_info">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Born_info">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Born</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Today_info">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Today</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Total_Minute_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>150</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Minute</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Total_Hour_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Hour</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Total_Second_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>190</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Second</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Total_Days_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>230</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Days</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Next_Birthday_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>390</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Birthday</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Total_Months_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>310</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Months</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Born_Day_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Born Day</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Total_Weeks_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>270</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Weeks</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Next_Born_Day_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>430</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Birthday Day </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Next_Born_Day_info_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>470</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Birthday Day </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Total_Hour">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>110</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Total Hour</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Total_Minute">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>150</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Total Minute</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Total_Days">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>230</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Total Days</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Total_Second">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>190</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Total Second</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Res_Born_Day">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>350</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Born Day</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Total_Months">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>310</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Total Months</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Next_BirthDay">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>390</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Next Birthday</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Total_Weeks">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Total Weeks</string>
    </property>
   </widget>
   <widget class="QLabel" name="Today_Year_info">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Next_Born_Day_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>470</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Next_Born_Day">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>430</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="Age_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Age">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLabel" name="Results">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Results</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Next_Year_info">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Different_info">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Different</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Different_Month">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Different Month</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Different_Year">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Different Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Different_Day">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Different Day</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Horoscope">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Horoscope</string>
    </property>
   </widget>
   <widget class="QLabel" name="Horoscope_info">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Horoscope</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
